weave-js/src/components/Panel2/PanelFacet/common.tsx

```diff
@@ -191,6 +207,16 @@ export const PanelFacetConfig: React.FC<PanelFacetProps> = props => {
   return (
     <ConfigPanel.ConfigSection label={`Properties`}>
       {dashboardConfigOptions}
+      <ConfigPanel.ConfigOption label={'x-axis-label'}>
+        <ConfigPanel.TextInputConfigField
+          dataTest="x-axis-label-input"
+          label="x-axis label"
```

Comment:
the dash and lower case looks odd to me

Reply:
synced with yangyi we going to go with "X-axis label" and just upper case X and Y